### PR TITLE
Table: add csv output

### PR DESCRIPTION
**Pull request by matthiasdiener**

### CODE REVIEW

File: pytools/__init__.py [hunk 1]
```diff
@@ -1597,6 +1597,32 @@ def github_markdown(self):
 
         return "\n".join(lines)
 
+    def csv(self, dialect='excel', **fmtparams):
```
Comment (inducer) [inline pytools/__init__.py:1600]: Maybe make `fmt_params` a separate argument? Otherwise we'll have a bear of a time extending this interface later.

Comment (inducer) [inline pytools/__init__.py:1600]: Document `fmt_params` (or maybe `csv_kwargs`), e.g. by linking to `:func:\`csv.writer\``.

Comment (matthiasdiener) [reply pytools/__init__.py]: Fixed (if I understood your comment correctly).

Comment (matthiasdiener) [reply pytools/__init__.py]: Added.

**Comment by inducer** [comment]
LGTM. Thanks for your contribution!

**Pull request merged**